server/test/services/applicant/ApplicantServiceTest.java

```diff
@@ -2702,7 +2702,7 @@ public void relevantProgramsForApplicant_withApplicationStatusAndOlderProgramVer
 
     // Publish a new program that has an updated set of status configurations that doesn't include
     // the application's status. The displayed status definition configuration should be pulled
-    // from the program version associated with the application.
+    // from the latest program version's status configuration
```

Comment:
This comment shouldn't reference the program version at all, right? Just `the program's ACTIVE status definitions` or similar?

Reply:
done!